internal/venice-test-common/src/main/java/com/linkedin/venice/utils/DataProviderUtils.java

```diff
@@ -91,6 +92,11 @@ public static Object[][] amplificationFactor() {
     return new Object[][] { { 1 }, { 3 } };
   }
 
+  @DataProvider(name = "Chunking-And-Partition-Counts")
+  public static Object[][] numberOfPartitions() {
```

Comment:
Let's rename this function to something more descriptive

Reply:
Ack

Reply:
Changed it to `chunkingAndPartitionCountsCombination`. LMK if that sounds ok?